changelog/v6.md

```diff
@@ -1,3 +1,9 @@
+## 6.2.2 (Oct 16th, 2023)
+
+### Added
+
+- An error, i.e. global is undefined is fixed which was occuring on browser console while running the plugin.
```

Comment:
Please change the date and modify this to:

### Fixed
- `global is not defined` error in the browser environment by providing a polyfill in webpack configuration